components/banner/Banner.js

```diff
@@ -0,0 +1,68 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Island from '../island';
+import Section from '../section';
+import { IconButton } from '../button';
+import cx from 'classnames';
+import theme from './theme.css';
+import { IconCloseMediumOutline } from '@teamleader/ui-icons';
+
+class Banner extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    color: PropTypes.oneOf(['white', 'neutral', 'mint', 'violet', 'ruby', 'gold', 'aqua']),
+    icon: PropTypes.element,
+    onClose: PropTypes.function,
+    fullWidth: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    color: 'white',
+  };
+
+  constructor() {
+    super(...arguments);
+    this.handleOnClick = ::this.handleOnClick;
```

Comment:
Wording seems a little off here, `on` and `handle` combined seems weird. 

**PP:** Use `handleClick` 😀

Reply:
Oopsie doopsie

 :ok: by @lowiebenoot on 26/10/2017 11:55:24